Domain/Projects/Project.cs

```diff
@@ -6,16 +7,40 @@
 
 namespace Domain.Projects
 {
-    public class Project
+    public class     Project
     {
         // Added by NG
         // Modified by XV & NVZ & EKT & DM
         public int Id { get; set; }
+
+        public string Title { get; set; }
+
+        public DateTime StartDate { get; set; }
+
+        public DateTime EndDate { get; set; }
+
+
+        private List<Phase> _phases;
+
+        public List<Phase> Phases
+        {
+            get
+            {
+                if (_phases == null)
+                {
+                    _phases = new List<Phase>();
+                }
+
+                return _phases;
+            }
+            set => _phases = value;
+        }
+
         public Platform Platform { get; set; }
         public UIMVCUser User { get; set; }
-        public string Title { get; set; }
+       
         public string Goal { get; set; }
-        public string Status { get; set; }
+        public string Status { get; set; } //TODO bool? 
```

Comment:
Reeds besproken: bool is niet modulair. Zowel enum als string (zoals wij hier doen bvb. OPEN, GESLOTEN, NIET ZICHTBAAR, ...)